test/WebJobs.Script.Tests.Integration/TestScripts/CSharp/TwilioReference/run.csx

```diff
@@ -1,8 +1,10 @@
-#r "Twilio.Api"
+#r "Twilio"
 
-using Twilio;
+using Twilio.Rest.Api.V2010.Account;
+using Twilio.Types;
 
 public static void Run(string input, TraceWriter log)
 {
     log.Info(input);
+    var message = new CreateMessageOptions(new PhoneNumber("+1704XXXXXXX"));
```

Comment:
Can you move this line above the `log.Info()`? That would guarantee that it doesn't throw an exception. If we see the test data in the logs, it means the call to instantiate the message succeeded.